ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusTextMessage.qml

```diff
@@ -69,12 +69,16 @@ Item {
         readonly property int effectiveHeight: d.veryLongChatText && !d.readMore
                                                ? Math.min(chatText.implicitHeight, 200)
                                                : chatText.implicitHeight
+        property FontMetrics metrics: FontMetrics {
```

Comment:
Do we need to expose it as property?

Reply:
No problem functionally but if it is used only internally then it should be private, it will be easier to grasp the intention when reading the code.